internal/fsutil/create.go

```diff
@@ -121,6 +125,28 @@ func createSymlink(o *CreateOptions) error {
 	return os.Symlink(o.Link, o.Path)
 }
 
+func createHardLink(o *CreateOptions) error {
+	debugf("Creating hard link: %s => %s", o.Path, o.Link)
+	targetInfo, err := os.Lstat(o.Link)
+	if err != nil && os.IsNotExist(err) {
+		return fmt.Errorf("the target file does not exist: %s", o.Link)
```

Comment:
Once we have the tests in place we can see how the errors look like and if they match the style of `create_test.go`.

Reply:
Added in [a8bd58e](https://github.com/letFunny/chisel/pull/17/commits/a8bd58e14f97611cbda8d3bb02f7494c2c25daa5) and [c8a7931](https://github.com/letFunny/chisel/pull/17/commits/c8a79317db2984e19b09b5386932d03224de9de2).